cli/src/semgrep/lsp/server.py

```diff
@@ -33,7 +33,7 @@ def __init__(self) -> None:
         self.polling: bool = True
 
     def start_ls(self) -> None:
-        cmd = ["osemgrep", "lsp"]
+        cmd = ["osemgrep", "lsp", "--experimental"]
```

Comment:
This need to be "semgrep", not "osemgrep", because osemgrep is not part of the pip package nor Docker image.

Reply:
In fact I don't even know how this can work ..., what is the path of this osemgrep and how our test infrastructure can find this osemgrep.

Reply:
we're not calling osemgrep, we're calling semgrep-core and passing the above as arguments. I'll rename `cmd` to `args`. This is functioning the same as `call_osemgrep`, but looks different because we're using Popen and not os.exec here so we can keep it as a child process